nonlin_sym.py

```diff
@@ -241,10 +242,14 @@
 # point fixed on the ground
 o = mec.Point('o')
 
+# point fixed on the kickplate (laterally moving ground)
+p = mec.Point('p')
+p.set_pos(o, y*N['2'])
+
 # rear wheel contact point, moves in ground plane, y is the kickplate lateral
 # location
 nd = mec.Point('nd')
-nd.set_pos(o, q1*N['1'] + q2*N['2'])
+nd.set_pos(p, q1*N['1'] + (q2 - y)*N['2'])
```

Comment:
```suggestion
nd.set_pos(o, q1*N['1'] + (q2 - y)*N['2'])
```

is this supposed to be o?

Reply:
It does not work for me. 
Error:
    raise PrintMethodNotImplementedError("Unsupported by %s: %s" % (str(type(self)), str(type(expr))) + \
sympy.printing.codeprinter.PrintMethodNotImplementedError: Unsupported by <class 'sympy.printing.numpy.SciPyPrinter'>: <class 'sympy.core.function.Derivative'>
Set the printer option 'strict' to False in order to generate partially printed code.

Still to investigate it

Reply:
What doesn't work for you?

Reply:
Does it work for you? It gives me error on eval_dynamic = sm.lambdify([qs, us, fs, rs, ps], [A_all, b_all], cse=True), that I am  not able to solve so far

Reply:
The current state of this pull request runs fine for me using the conda environment in the bicycle-kickplate-model-env.yml.

Reply:
I just checked this and I think it is correct. I use q1 and q2 to locate the wheel contact point relative to the origin O. So that means that the absolute position of the wheel contact is at a lateral distance q2.